src/test/java/uk/gov/hmcts/reform/migration/service/ApplicantDataMigrationServiceTest.java

```diff
@@ -0,0 +1,229 @@
+package uk.gov.hmcts.reform.migration.service;
+
+import com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.domain.Applicant;
+import uk.gov.hmcts.reform.domain.OldApplicant;
+import uk.gov.hmcts.reform.domain.common.Address;
+import uk.gov.hmcts.reform.domain.common.EmailAddress;
+import uk.gov.hmcts.reform.domain.common.MobileNumber;
+import uk.gov.hmcts.reform.domain.common.ApplicantParty;
+import uk.gov.hmcts.reform.domain.common.TelephoneNumber;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ApplicantDataMigrationServiceTest {
+    private final String APPLICANT = "Applicant name";
+    private final String EMAIL = "email@email.com";
+    private final String MOBILE = "07778987656";
+    private final String COUNTY = "County";
+    private final String COUNTRY = "Country";
+    private final String POSTCODE = "BT17NT";
+    private final String POSTTOWN = "Ballymena";
+    private final String ADDRESSLINE1 ="ADDRESSLINE1";
```

Comment:
Why these names are without any separator? They are just hard to read. Use `_` as a separator next time.

Reply:
👍

Reply:
made change